source/frontend/assets/components/RecipeDisplay.js

```diff
@@ -496,25 +500,23 @@ class RecipeDisplay extends HTMLElement {
    */
   bindCompleteButton(data) {
     const btn = this.shadowRoot.getElementById('made-this-button');
-    btn.addEventListener('click', () => {
+    btn.addEventListener('click', async () => {
       RecipeDisplay.jSConfetti.addConfetti({ emojis: ['🥵', '🔥', '🌶️'] });
-      if (data.completed === false) {
-        database.completeRecipe(data);
-        const newBox = document.createElement('completed');
-        newBox.innerHTML = 'Completed!';
-        newBox.innerHTML += '<br><br>Upload a picture of your reaction!';
-        btn.parentElement.appendChild(newBox);
-        btn.parentElement.removeChild(btn);
-        const imgPreview = this.shadowRoot.getElementById('imgPreview');
-        imgPreview.style.display = '';
-        const uploadImg = this.shadowRoot.getElementById('imgUpload');
-        uploadImg.style.display = '';
-        const submitBtn = this.shadowRoot.getElementById('submitButton');
-        submitBtn.style.display = '';
-        this.GetImgurImage();
-        this.json.reactions = 'assets/images/placeholder.png';
-        this.SubmitReaction();
-      }
+      await database.completeRecipe(data);
```

Comment:
I will make another PR to remove this and add the frontend for displaying a completed recipe.

Reply:
I am not sure what you mean by that. This is the function that gets called when the complete button is pressed, it has nothing to do with displaying a completed reecipe or removing it won't help with that in anyway.

Reply:
I selected the wrong lines, i meant the lines below where it shows the input for an uploading image.

Reply:
ok